SimpleInventoryManagement/Util/ConsoleColorType.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace SimpleInventoryManagement.Util
+{
+    /**
+     * This is an enumeration to specify type 
+     * of output, each type given a console color. 
+     */
+    public enum ConsoleColorType
+    {
+        Information = ConsoleColor.Blue,
+        Error = ConsoleColor.Red,
+        Warning = ConsoleColor.DarkYellow,
+        Prompt = ConsoleColor.Magenta,
+        Success = ConsoleColor.DarkGreen,
+        Failure = ConsoleColor.DarkGray,
+        Default = ConsoleColor.White
+    }
+}
```

Comment:
@tjaber-fts 
I have addressed your comment, and pushed the changes, 
please let me know if anything else need to be adjusted.